api/routes/graphs.py

```
@@ -225,3 +227,46 @@ async def delete_graph(request: Request, graph_id: str):
+async def get_graph_user_rules(request: Request, graph_id: str):
+    """Get user rules for the specified graph."""
+    try:
+        full_graph_id = _graph_name(request.state.user_id, graph_id)
+        logging.info("Getting user rules for graph: %s (full: %s)", graph_id, full_graph_id)
+        user_rules = await get_user_rules(full_graph_id)
+        logging.info("Retrieved user rules length: %d", len(user_rules) if user_rules else 0)
+        logging.info("Retrieved user rules content: %s", repr(user_rules))
+        return JSONResponse(content={"user_rules": user_rules})
+    except GraphNotFoundError:
+        return JSONResponse(content={"error": "Database not found"}, status_code=404)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)